File: HM326Project/pinyougou_parent/pinyougou_interface/src/main/java/cn/itcast/core/service/content/ContentService.java
package cn.itcast.core.service.content;

import java.util.List;

import cn.itcast.core.pojo.ad.Content;
import com.github.pagehelper.PageInfo;


public interface ContentService {

	public List<Content> findAll();

	public PageInfo<Content> findPage(Content content, Integer pageNum, Integer pageSize);

	public void add(Content content);

	public void edit(Content content);

	public Content findOne(Long id);

	public void delAll(Long[] ids);

	List<Content> findByCategoryId(long categoryId);

}

File: HM326Project/pinyougou_parent/pinyougou_web_portal/src/main/webapp/js/service/contentService.js
app.service("contentService",function($http){
    this.findByCategoryId = function(categoryId){
        return $http.get("content/findByCategoryId.do?categoryId="+categoryId);
    }




    //查询商品分类信息
    this.findItemCatList = function () {

        return $http.get("content/findItemCatList.do");
    }

});